lightwood/encoders/time_series/helpers/transformer_helpers.py

```diff
@@ -0,0 +1,78 @@
+import math
+import torch
+import torch.nn as nn
+
+
+def len_to_mask(lengths, zeros):
+    # Clean trick from:
+    # https://stackoverflow.com/questions/53403306/how-to-batch-convert-sentence-lengths-to-masks-in-pytorch
+    mask = torch.arange(lengths.max())[None, :] < lengths[:, None]
+    if zeros:
+        mask = ~mask  # Logical not
```

Comment:
Might be worth being explicit here (in a comment and/or `dtype` cast if `mask` is intended to have type bool or int since bitwise not doesn't operate on `0` and `1` the same way it would operate on `False` and `True` (I assume the above code is meant for dtype==bool, but maybe I'm wrong. hence why a comment would help)

Reply:
The mask has a `bool` type, as it is created by a comparison operator. I will add a comment and proper documentation to this and to other functions (I forgot to write the docs before the PR)